File: Modulo15/Aviao-C++/Aviao.cpp
#include "Aviao.h"
#include <iostream>
using namespace std;

namespace poo {

    void Aviao::decolar() {
        cout << "Liga as turbinas e levanta voo" << endl;
    }

    void Aviao::aterissar() {
        cout << "Desce o trem de pouso e freia" << endl;
    }

    void Aviao::subir() {
        cout << "Faz check-in e embarca" << endl;
    }

    void Aviao::descer() {
        cout << "Desembarca e pega as malas" << endl;
    }
    
}

File: Modulo11/Aviao-CSharp/Aviao.cs
using System;

namespace poo {
    
    public class Aviao : Voador, MeioDeTransporte{
        public void decolar() {
            Console.WriteLine("Liga as turbinas e levanta voo");
        }
        
        public void aterissar() {
            Console.WriteLine("Desce o trem de pouso e freia");
        }

        public void subir() {
            Console.WriteLine("Faz check-in e embarca");
        }

        public void descer() {
            Console.WriteLine("Desembarca e pega as malas");
        }
    }
}

